protocol-concepts/DynamicFees.md

```diff
@@ -0,0 +1,382 @@
+# Introduction to Dynamic Fees
+
+Uniswap v4 introduces dynamic fees, allowing for flexible and responsive fee structures managed through hooks. This feature enables pools to adapt fees to changing market conditions, potentially improving liquidity provider profitability and overall market efficiency.
+
+## What are Dynamic Fees?
+
+Unlike the static fee tiers in Uniswap v3 (0.05%, 0.30%, 1.0%) or the single fee in v2, dynamic fees in v4 can:
```

Comment:
I think we add clarity that Dynamic Feess are fees paid by swappers, that accrue to liquidity providers.

it adds more clarity around where does dynamic fee fit within: LPFee (swap fee), protocol fee, and hook fee